start.sh

```diff
@@ -2,6 +2,11 @@
 
 pidfile=/app/tmp/pids/server.pid
 
+if [ ! -f "./config/initializers/recaptcha.rb" ]; then
+    echo -e "\e[33mWARNING\e[0m: You haven't configured recaptcha!"
+    echo -e "\e[94mRead More\e[0m: https://github.com/publiclab/mapknitter#installation-steps"
+fi
+
```

Comment:
I would have done something like this, maybe we should just configure it, and skip the overhead the developer would have of just copying and pasting a file.

![image](https://user-images.githubusercontent.com/19597045/71414881-cb0e7300-2637-11ea-88f6-90d1e2b6452f.png)